### ISSUE TITLE
Dropdown Input - The 'The input element does not have an associated visible label' accessibility error occurs - The 'Select...' label should not be read by a scanner

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7071]
T14862 - Accessibility Issue: Dropdown must have an associated visible label.
https://surveyjs.answerdesk.io/internal/ticket/details/T14862
